scripts/factory/get_ft_balance_from_factory.cdc

```diff
@@ -0,0 +1,13 @@
+import "FungibleToken"
+import "ExampleToken"
+
+import "FTBalanceFactory"
+
+pub fun main(addr: Address) {
+    let acct = getAuthAccount(addr)
+    let ref = &acct as &AuthAccount
+
+    let factory = FTBalanceFactory.Factory()
+
+    let provider = factory.getCapability(acct: ref, path: ExampleToken.VaultPublicPath) as! Capability<&{FungibleToken.Balance}>
```

Comment:
Do we need to check the Capability within the script for us to be sure of the Capability validity?

Reply:
You're right, the scope of concern here is with retrieval from the factory. I don't think it's necessary to check the Capability.